app/components/festivals/forms/new-festival.tsx

```diff
@@ -0,0 +1,244 @@
+    mapsVersion: z.enum(['v1', 'v2', 'v3']).default('v1'),
+    publicRegistration: z.boolean().default(false),
+    eventDayRegistration: z.boolean().default(false),
+    festivalType: z.enum(['glitter', 'twinkler', 'festicker']).default('glitter'),
+
+    description: z.string().optional(),
+    address: z.string().optional(),
+    locationLabel: z.string().optional(),
+    locationUrl: z.string().url().optional().or(z.literal('')),
+    startDate: z.coerce.date().optional(),
+    endDate: z.coerce.date().optional(),
+    generalMapUrl: z.string().optional(),
+    mascotUrl: z.string().optional(),
+    illustrationPaymentQrCodeUrl: z.string().optional(),
+    gastronomyPaymentQrCodeUrl: z.string().optional(),
+    entrepreneurshipPaymentQrCodeUrl: z.string().optional(),
+    illustrationStandUrl: z.string().optional(),
+    gastronomyStandUrl: z.string().optional(),
+    entrepreneurshipStandUrl: z.string().optional(),
+    festivalCode: z.string().optional(),
+    festivalBannerUrl: z.string().optional(),
+});
+
+export default function NewFestivalForm({ onSuccess }: { onSuccess: () => void }) {
+    const router = useRouter();
+    const form = useForm<z.infer<typeof FormSchema>>({
+        resolver: zodResolver(FormSchema),
+        defaultValues: {
+            name: "",
+            status: "draft",
+            mapsVersion: "v1",
+            publicRegistration: false,
+            eventDayRegistration: false,
+            festivalType: "glitter",
+        }
+    });
+
+    const onSubmit = form.handleSubmit(async (data) => {
+        const festivalData = {
+            ...data,
+            createdAt: new Date(),
+            updatedAt: new Date(),
+            reservationsStartDate: new Date(),
+        };
+
+        const result = await createFestival(festivalData);
+
+        if (result.success) {
+            toast.success(result.message);
+            onSuccess();
+            router.push("/dashboard/festivals");
+        } else {
+            toast.error(result.message);
+        }
+    });
+
+    return (
+        <Form {...form}>
+            <form onSubmit={onSubmit} className="space-y-6 max-w-3xl">
+                {/* Basic Information Section */}
+                <div className="space-y-4 p-4 border rounded-lg">
+                    <h3 className="font-medium">Informacion Básica</h3>
+
+                    <FormField
+                        control={form.control}
+                        name="name"
+                        render={({ field }) => (
+                            <FormItem>
+                                <FormLabel>Nombre de Festival</FormLabel>
+                                <FormControl>
+                                    <Input {...field} />
+                                </FormControl>
+                                <FormMessage />
+                            </FormItem>
+                        )}
+                    />
```

Comment:
Hay unos componentes reutilizables en la carpeta `app/components/form/fields` que podrías usar acá y en cualquier otro form

```suggestion
                      <TextInput
                        formControl={form.control}
                        name="name"
                        label="Nombre de Festival"
                      />
```